packages/bus/src/dispatchers/HttpCommandDispatcher.ts

```diff
@@ -12,6 +12,8 @@ export class HttpCommandDispatcher implements CommandDispatcher {
   private readonly client: RateLimitedAxiosInstance;
 
   constructor(
+    @inject(RetryStrategy)
+    private readonly retryStrategy: RetryStrategy,
```

Comment:
You don't have any suitable retry strategy that can catch and handle HTTP exceptions. `ExponentialBackoffRetryStrategy` is going to work only with AMQP